src/tachyons-base.css

```diff
@@ -31,3 +31,14 @@ a {
 code {
   color: #59cd90; /* green */
 }
+
+
+/* Max-width */
+.eh-mw9 {
+  max-width: 76rem;
+}
```

Comment:
Seems that this should go with the other max width classes:
![image](https://cloud.githubusercontent.com/assets/5497885/25147375/8e202ebe-2434-11e7-937b-3e5ae31442d7.png)